src/atoms/Link/Link.web.js

```diff
@@ -1,3 +1,50 @@
-const Link = () => null;
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import { getColor } from '../../_helpers/theme';
+
+const StyledText = styled(Text)`
+  color: ${(props) => getColor(props.theme, 'primary.800')};
+  text-decoration: none;
+  cursor: pointer;
+  &:hover {
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.800')};
+  }
+  &:focus {
+    outline: none;
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.900')};
+  }
+  &:active {
+    outline: none;
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.700')};
+  }
+  &:visited {
+    outline: none;
+    color: ${(props) => getColor(props.theme, 'primary.700')};
+  }
+`;
+
+const Link = ({ size, children, href }) => {
+  return (
+    <StyledText as="a" size={size} href={href}>
+      {children}
+    </StyledText>
+  );
+};
+
+Link.propTypes = {
+  children: PropTypes.string.isRequired,
+  size: PropTypes.oneOf(['xsmall', 'small', 'medium', 'large']),
+  href: PropTypes.string,
```

Comment:
need to add `target` prop as well. I completely forgot about that

Reply:
thought of it. but I didn't add keeping API thing in mind. 
also should I add href and target on stories too ?

Reply:
👆  Yes.